examples/crowd-funding/src/lib.rs

```diff
@@ -22,7 +256,7 @@ impl ContractAbi for CrowdFundingAbi {
 }
 
 impl ServiceAbi for CrowdFundingAbi {
-    type Parameters = ApplicationId<fungible::FungibleTokenAbi>;
+    type Parameters = String;
```

Comment:
This is definitely a regression.
We go from typed parameters to untyped ones.
Is that really necessary? Is there any other way.

Reply:
You're right, once https://github.com/linera-io/linera-protocol/pull/861 is merged we can probably revert that change.

(It was necessary because having the user specify the fungible token application ID both in BCS hex format _and_ in structured JSON format made it almost unusable.)

Reply:
So, 861 is merged.

Reply:
Yes, already reverted the `String` change.